src/applications/vaos/utils/selectors.js

```diff
@@ -290,7 +290,7 @@ export function getChosenClinicInfo(state) {
   const typeOfCareId = getTypeOfCare(data)?.id;
   return (
     clinics[`${data.vaFacility}_${typeOfCareId}`]?.find(
-      clinic => clinic.clinicId === data.clinicId,
+      clinic => clinic.id.split('_')[1] === data.clinicId,
```

Comment:
Wouldn't this be `clinic => clinic.id === data.clinicId`? I think the ids we put in `clinicId` would be the full id: https://github.com/department-of-veterans-affairs/vets-website/pull/13047/files#diff-cb8a75225ac0864b221538caaebc1c27R681

Reply:
The transformer is formatting the clinic id like this:

id: `var${clinic.siteCode}_${clinic.clinicId}`

Reply:
Sure, but `data.clinicId` should also be in that same format, since it's form data that uses an `enum` array based on the clinic ids (see the line I linked).